### DISCUSSION TITLE
Is it possible to sync a collection with Instagram feed?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#860] [Q&A]
Hi

Does anyone know a trick to insert instagram content with mavo?
Or if there is a way to sync a mavo collection with Instagram feeds?

**Comment by DmitrySharabin** [comment]
Lea developed [the app](https://forkgasm.com/restaurants/) that uses the photo she uploads to Instagram. But AFAIK, she uses the dedicated script to get them (you can explore [the app repo](https://github.com/leaverou/forkgasm)). So I don't know whether it's possible to get an Instagram feed with Mavo.

What do you expect this app to do? It might be other people need it too, and we could try to build a plugin for it. You can request it [here](https://github.com/mavoweb/plugins/issues).

**Comment by GalinhaLX** [comment]
I want to display a instagram feed into my mavo page

Something like this:
![image](https://user-images.githubusercontent.com/89095737/172920066-44332ef6-01ad-40b8-bb0c-a11446f52e62.png)

or

![image](https://user-images.githubusercontent.com/89095737/172919248-9f72b025-3524-48dc-b410-064b4cada3cc.png)


instagram allows you to copy/paste each  post link which will be like an image of the post that you can embed into yout website, but you need to do it manually, for each post :(

There are some 'code' (like instafeed.js (not sure it still works because of last Instagram API) which allows you to just connect to a instagram feed, and allow display of last X posts related to a special hastag for example:
much more user friendly

Mavo could connect to an instagram feed, and allow to filter or display it with specific styles

**Reply by DmitrySharabin** [reply]
I see.

I just remembered about one interesting project that might be useful in your case. It's called [Shapir](https://shapir.org) and is led by Tarfah Alrashed (@tarfahalrashed). It might be she could help you and / or your use case might interest her. Or, best of all, Tarfah already has a solution for you. ;)

**Reply by GalinhaLX** [reply]
Very interesting ! I remember to see this once (before mavo-shapir maybe 

Maybe this tool could be added on plugins documentation, if it is allowed?

Are there other un-official plugins or projects that connect with Mavo ? ;)
